src/w3c/headers.js

```diff
@@ -269,6 +270,12 @@ export function run(conf) {
   if (document.title && conf.isPreview && conf.prNumber) {
     document.title = `Preview of PR #${conf.prNumber}: ${document.title}`;
   }
+
+  if (conf.level) {
```

Comment:
We probably want to check that level is a decimal, and show an error if it's not.

Reply:
Just noting that we've had Level 0 specs, so we should allow that... basically any sequence of decimals should be fine.

Reply:
To clarify - by `decimal` you mean that it is okay to have the level be something like `2.5`, `2.00`, etc, yes?

Reply:
Probably only zero and positive integers?